pkgs/servers/sql/postgresql/default.nix

```diff
@@ -70,7 +70,8 @@ let
           # Remove static libraries in case dynamic are available.
           for i in $out/lib/*.a; do
             name="$(basename "$i")"
-            if [ -e "$lib/lib/''${name%.a}.so" ] || [ -e "''${i%.a}.so" ]; then
+            ext="${stdenv.hostPlatform.extensions.sharedLibrary}"
+            if [ -e "$lib/lib/''${name%.a}$ext" ] || [ -e "''${i%.a}$ext" ]; then
```

Comment:
You can remove this conditional with ```rm -f``` I think.

Reply:
I'm quite convinced that the conditional code needs to stay:
```
$ nix-build -A postgresql -A postgresql.lib`
# -snip-
$ ls result/lib/*.a
result/lib/libpgcommon.a  result/lib/libpgfeutils.a  result/lib/libpgport.a
```
Which would be removed if I dropped the conditionals and used `rm -f`.